elasticapm/base.py

```diff
@@ -103,6 +103,15 @@ def __init__(self, config=None, **inline):
             config.disable_send = True
         self.config = VersionedConfig(config, version=None)
 
+        # Insert the log_record_factory into the logging library
+        # The LogRecordFactory functionality is only available on python 3.2+
+        if sys.version_info >= (3, 2) and self.config.disable_log_record_factory is False:
```

Comment:
You're right, `not` would also work fine. I'm in the habit of `is <bool>` ever since I learned how much faster it is than `== <bool>`. But `not` is more readable and just as fast.

(I didn't realize we were 3.4+, good to know!)

Reply:
Done.